pyoverkiz/models.py

```diff
@@ -316,7 +316,12 @@ def __len__(self) -> int:
 class Command(dict):
     """Represents an OverKiz Command."""
 
-    def __init__(self, name: str, parameters: list[str] | None = None, **_: Any):
+    name: str
+    parameters: list[str | int | float] | None
```

Comment:
Why?

Reply:
Why I typed this? Or why I added int/float to the list?

I added int/float because we pass that to the API directly from ha-tahoma, and we don't case it to string first.